zokrates_cli/examples/book/assembly/bool_to_field.zok

```diff
@@ -0,0 +1,8 @@
+from "EMBED" import bool_to_field;
+
+def main(bool x) -> field {
+    // `x` is constrained by the compiler automatically so we can safely
```

Comment:
This always the case, bool_to_field is always free and safe, it can be removed from the embeds. This test is not necessary.